shardingsphere-integration-test/shardingsphere-proxy-docker-build/src/main/resources/tool/wait-for-it.sh

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/env bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#   Use this script to test if a given TCP host/port are available
+
+WAITFORIT_cmdname=${0##*/}
+
+echoerr() { if [[ $WAITFORIT_QUIET -ne 1 ]]; then echo "$@" 1>&2; fi }
+
+usage()
+{
+    cat << USAGE >&2
```

Comment:
I don't think you can simply add a license header when you copy a file from others, especially it's not licensed with AL2.0

https://github.com/vishnubob/wait-for-it/blob/master/wait-for-it.sh

Reply:
FYI @tristaZero

Reply:
@kezhenxu94 Hi, thanks for your kind remainder, and sorry for my lax checking. :)

Reply:
ok, I'll revise it later, it's better wget it in docker file

Reply:
fix it at #5765